src/main/resources/Jetbrains_Gray.theme.json

```diff
@@ -0,0 +1,121 @@
+{
+  "name": "JetBrains Gray",
+  "dark": false,
```

Comment:
gray mode  is part of dark mode, Its our way of saying lighter dark mode, so it should be true